docs/gitbook/tutorials/rollout-weights.md

```diff
@@ -0,0 +1,38 @@
+# Linkerd Rollout Weights
```

Comment:
Can you please move this to the Linkerd tutorial

Reply:
I believe this should be applicable to any rollout, not only Linkerd. What about removing `Linkerd` from the title?

Reply:
Ok then it could go under https://github.com/weaveworks/flagger/blob/master/docs/gitbook/usage/deployment-strategies.md#canary-release

Reply:
Done. I also added test to cover new functionality.